Solutions/GOVINDDIXIT/Task4/sine_curve.py

```diff
@@ -0,0 +1,9 @@
+import matplotlib.pyplot as plt
```

Comment:
Remove this file. Your task 4 PR is pending.

Reply:
I had send the pr from different branches